cmake/modules/extensions.cmake

```diff
@@ -1154,6 +1154,7 @@ endfunction(zephyr_check_compiler_flag_hardcoded)
 #    DATA_SECTIONS Inside the RAMABLE_REGION GROUP, initialized.
 #    RAMFUNC_SECTION Inside the RAMFUNC RAMABLE_REGION GROUP, not initialized.
 #    NOCACHE_SECTION Inside the NOCACHE section
+#    PM_SECTIONS   For PM related symbols
```

Comment:
but where is it placed ?

Inside or outside any existing sections ?

Reply:
Outside is probably the best, though I thought it could be target specific since it is tight to soc power management specific needs.